src/main/java/uk/ac/ebi/eva/pipeline/io/mappers/AnnotationLineMapper.java

```diff
@@ -183,26 +189,19 @@ private Integer parseStringInterval(String stringInterval) {
      *
      * The parseFrequencies option has been removed
      */
-    private void parseExtraField(ConsequenceType consequenceType, String extraField, VariantAnnotation currentAnnotation) {
-
+    private void parseExtraField(ConsequenceType consequenceType, String extraField) {
         for (String field : extraField.split(";")) {
             String[] keyValue = field.split("=");
 
             switch (keyValue[0].toLowerCase()) {
                 case "biotype":
                     consequenceType.setBiotype(keyValue[1]);
                     break;
-                case "hgvsc":
-                    currentAnnotation.getHgvs().add(keyValue[1]);
-                    break;
-                case "hgvsp":
-                    currentAnnotation.getHgvs().add(keyValue[1]);
-                    break;
                 case "polyphen": // Format is PolyPhen=possibly_damaging(0.859)
-                    consequenceType.addProteinSubstitutionScore(parseProteinSubstitutionScore("Polyphen", keyValue[1]));
+                    consequenceType.setPolyphen(parseProteinSubstitutionScore(keyValue[1]));
                     break;
                 case "sift": // Format is SIFT=tolerated(0.07)
-                    consequenceType.addProteinSubstitutionScore(parseProteinSubstitutionScore("Sift", keyValue[1]));
+                    consequenceType.setSift(parseProteinSubstitutionScore(keyValue[1]));
```

Comment:
I remember discussing this with @cyenyxe but unfortunately I didn't document the conclusion. Looking at the VEP doc it looks that there is only one Score and one Poly.. per Consequence. See [http://www.ensembl.org/info/docs/tools/vep/vep_formats.html#output](http://www.ensembl.org/info/docs/tools/vep/vep_formats.html#output)